src/transports/PostMessageIframeTransport.ts

```diff
@@ -5,10 +5,12 @@ class PostMessageIframeTransport extends Transport {
   public uri: string;
   public frame: undefined | null | Window;
   public postMessageID: string;
+  public origin: string;
 
   constructor(uri: string) {
```

Comment:
we'll still need the origin to come from the constructor here. We have to leave the origin to be specified by the consumers here, by passing it into the constructor. This is because we can't make an assumption about where they'd like the post message communication to come from. 

In a dev case for instance they may want *, in a cross domain case they may want the origin to come from a window they launched that has a separate domain they know.  If we lock them in it takes away the choice. 

The default to * after speaking with @shanejonas we both feel like the security change warrants a version that makes people explicitly state *, so they know what assumptions they're making about the transport.  

The pr looks great, and thanks for the questions and comments and getting this done!